kinbot/irc.py

```diff
@@ -121,7 +119,7 @@ def do_irc_calculations(self):
             if self.rxn.qc.qc == 'nwchem' and direction == 'Reverse':
                 direction = 'Backward'
 
-            odft = self.rxn.species.mult > 1
+            # odft = self.rxn.species.mult > 1
```

Comment:
Why is this line commented out?

Reply:
I commented it out because the variable isn't used anywhere in the code, I can uncomment it.